trick_sims/SIM_aircraft/models/Aircraft/include/Waypoint.hh

```diff
@@ -0,0 +1,36 @@
+/************************************************************************
+PURPOSE: (Handles the waypoints.)
+**************************************************************************/
+#ifndef Waypoint_HH
+#define Waypoint_HH
+
+#ifndef ARRAY_SIZE
+#define ARRAY_SIZE  10
+#endif
+
+#include <string>
+#include <vector>
+#include <fstream>
+#include <iostream>
+
+class WaypointList {
+    public:
+    double north[ARRAY_SIZE];
```

Comment:
Can we make this class a little more object-oriented? Make the arrays and length private members, and add getter and setter methods for them as needed, that way you could check length in this class instead of making the Aircraft class doing it. That way, if you later decide to do something like changing the arrays to use different data structure, you are able to do it by just changing this class and not changing Aircraft.

Reply:
I went ahead and made the variables private.